HpToolsLauncher/Runners/AlmTestSetsRunner.cs

```diff
@@ -1713,8 +1718,15 @@ private TestState GetTsStateFromQcState(string qcTestStatus)
                     return TestState.Running;
                 case "Success":
                 case "Finished":
-                case "FinishedPassed":
-                    return TestState.Passed;
+                case "FinishedPassed":
+					{
+                        if (qcTestStatus.Message.Contains("warning"))
+						{

```

Comment:
If the ALM returned test message contains "warning" the test has generated a warning, but from ALM is wrongly returned as PASSED, we should overwrite it.